src/khepri/rra/journey/assets/shell-components.css

```
@@ -198,6 +198,60 @@
+
+/*
+ * Available to the shell for the first time. `journey.css` has carried this since the upload form's
+ * file input; the frame needs it for the organization control's purpose text, and the shell does not
+ * load that stylesheet. `clip-rect` rather than `display: none` so the text reaches a screen reader.
+ */
+.visually-hidden {
+  position: absolute;
+  inline-size: 1px;
+  block-size: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)